src/univalue/configure.ac

```diff
@@ -15,7 +15,7 @@ m4_define([libunivalue_version], [libunivalue_major_version().libunivalue_minor_
 
 
 AC_INIT([univalue], [1.0.3],
-        [http://github.com/jgarzik/univalue/])
+        [https://github.com/jgarzik/univalue/])
```

Comment:
You can't modify our subtrees here, if you'd like you change this link you'll need to do it upstream: https://github.com/bitcoin-core/univalue.

Reply:
Ah, I had not realized these were subtrees. Thanks. I will do it in the upstream repos.

Reply:
https://github.com/jgarzik/univalue/pull/81